chopsticks/bajun.yml

```diff
@@ -0,0 +1,30 @@
+# In the future we want to add this config to https://github.com/AcalaNetwork/chopsticks/tree/master/configs.
+# But we should wait until we have the governance pallets included because this will change this config.
+endpoint:
+  - wss://rpc-parachain.bajun.network
+  - wss://bajun.api.onfinality.io/public-ws
+  - wss://bajun.public.curie.radiumblock.co/ws
+mock-signature-host: true
+block: ${env.BAJUN_PARACHAIN_BLOCK_NUMBER}
+db: ./chopsticks/bajun-parachain.db.sqlite
+runtime-log-level: 5
+# wasm-override: bajun_runtime.wasm
+
+# Once we have governance we want to make Alice single councillor and technical committee member.
+import-storage:
+  Sudo:
+    Key: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY # Alice
+  System:
+    Account: # Give our dev accounts some funds
+      - - - 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY # Alice
+        - providers: 1
+          data:
+            free: 1000000000000000
+      - - - 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty # Bob
+        - providers: 1
+          data:
+            free: 1000000000000000
+      - - - 5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y # Charlie
+        - providers: 1
+          data:
+            free: 1000000000000000
```

Comment:
Once governance stuff is on the live network, we will add some entries here that override the council and technical committee members with Alice, Bob and Charlie.